### ISSUE TITLE
Pygment version is too old

### ISSUE THREAD

**Issue by marcdexet-cnrs** [technige/py2neo#722]
Hi,

When you use py2neo with jupyter (as any data scientist), you have a ugly incompatibility with two tools

```
py2neo 4.1.2 has requirement prompt-toolkit==1.0.15, but you'll have prompt-toolkit 2.0.7 which is incompatible.
jupyter-console 6.0.0 has requirement prompt-toolkit<2.1.0,>=2.0.0, but you'll have prompt-toolkit 1.0.15 which is incompatible.
ipython 7.1.1 has requirement prompt-toolkit<2.1.0,>=2.0.0, but you'll have prompt-toolkit 1.0.15 which is incompatible.
```

**Comment by technige** [comment]
Looks like you mean `prompt-toolkit`, rather than `pygments` as per the title of this issue.

The reason this is pinned is because there were significant backwards-incompatible changes introduced when prompt-toolkit 2.0 was released:
https://python-prompt-toolkit.readthedocs.io/en/master/pages/upgrading.html#upgrading-2-0

So while this is a reasonable request, this is a non-trivial piece of work to undertake. In other words, it needs doing, but I don't know when I'll get around to it.

**Comment by technige** [comment]
This should be fixed in py2neo 4.1
